### ISSUE TITLE
can i connect my database ?

### ISSUE THREAD

**Issue by prameelareddi** [thongekchakrit/automated-data-analysis#3]
I want to see my database data via charts
Are there any possible chances to connect the database with chartAI ?

**Comment by thongekchakrit** [comment]
What type of databases would you like to connect to? If there's time I can probably help to implement the feature.

**Comment by prameelareddi** [comment]
Hello @thongekchakrit , Thank You for your response.....
What type of databases would you like to connect to? If there's time I can probably help to implement the feature.
I need to connect any relational database like Postgres,MySQL..etc
Great thanks !!

**Comment by kulpatel** [comment]
yes i too want it to connect to my postgressql and generate queries conidering schema in mind. Assume i hhave data of all the sales and inventory of mine in DB, now i wann see some chart i.e this months sales generated by so and so product. i should be able to see chart etc etc..some ffancy stuf. 1> key here is translate text to sql 2> get that sql data and plot a user inputed chart out of that. Possible?
